frontend/src/App.tsx

```diff
@@ -41,6 +41,7 @@ import {
   Elements,
   BlockElement,
   SimpleElement,
+  ElementWrapper,
```

Comment:
Rename to `ReportElement` so it's easy to remember that this is just `SimpleElement` + `reportId`

Reply:
done